docs/content/policy-reference.md

```diff
@@ -878,6 +879,27 @@ org_chart_permissions[entity_name]
 ```live:graph/reachable/example:output
 ```
 
+It may be useful to find all reachable paths from a root element. `graph.reachable_paths` can be used for this. Note that cyclical paths will terminate on the repeated node.
+
+```live:graph/reachable_paths/example:module
+package graph_reachable_paths_example
+path_data = {
+    "aTop": [],
+    "cMiddle": ["aTop"],
+    "bBottom": ["cMiddle"],
+    "dIgnored": []
+}
+all_paths[root] = paths {
+    path_data[root]
+    paths := graph.reachable_paths(path_data, {root})
+}
```

Comment:
[nit] some extra whitespace would make this look like the example above,
```suggestion
package graph_reachable_paths_example

path_data = {
    "aTop": [],
    "cMiddle": ["aTop"],
    "bBottom": ["cMiddle"],
    "dIgnored": []
}

all_paths[root] = paths {
    path_data[root]
    paths := graph.reachable_paths(path_data, {root})
}
```
Adding an example like this for a new built-in is great. We should do that more often (#3786).